modules/plugin/imagemosaic/src/test/resources/org/geotools/gce/imagemosaic/test-data/remote_test/remote_test.properties

```diff
@@ -0,0 +1,17 @@
+#-Automagically created from GeoTools-
+#Thu Mar 08 12:15:32 CET 2018
+MosaicCRS=EPSG\:4326
+Levels=0.057934032977228433,0.04039598061277999
+Heterogeneous=false
+Name=remote_test
+HeterogeneousCRS=false
+TypeName=remote_test
+Caching=false
+ExpandToRGB=false
+LocationAttribute=location
+PathType=CUSTOM
+SuggestedSPI=com.sun.imageio.plugins.png.PNGImageReaderSpi
+CheckAuxiliaryMetadata=false
+LevelsNum=1
+SuggestedFormat=org.geotools.gce.imagemosaic.remote.RemoteImageFormat
```

Comment:
So the test uses a ready-made property file. Would it work with the documented and recommended approach of starting from scratch and using a indexer.properties file instead, with no shapefile, no sample image and no mosaic property file?

Reply:
No that would not work indeed. There is no way to scan for images. The purpose is to make it possible to connect the imagemosaic to any remote raster reader, such as the S3 reader in particular. You are responsible yourself to make an index vector source with all the URLs and then you can build an image mosaic reader on top of that.